src/deepsparse/utils/onnx.py

```diff
@@ -128,13 +129,13 @@ def model_to_path(model: Union[str, Model, File]) -> str:
     return model
 
 
-def get_external_inputs(onnx_filepath: str) -> List:
+def get_external_inputs(onnx_model: Union[str, ModelProto]) -> List:
     """
     Gather external inputs of ONNX model
-    :param onnx_filepath: File path to ONNX model
+    :param onnx_model: File path to ONNX model or ONNX model object
     :return: List of input objects
     """
-    model = onnx.load(onnx_filepath)
+    model = onnx_model if isinstance(onnx_model, ModelProto) else onnx.load(onnx_model)
```

Comment:
```suggestion
    model = onnx_model if isinstance(onnx_model, ModelProto) else onnx.load(onnx_model, load_external_data=False)
```
Whenever we just take a look at inputs and derive something from them, without actually using the model's weights, let's load without external data to avoid loading big data to memory. This applies to all the functions edited in this PR.

Reply:
Good call. Applied 🙂